client/src/pages/SignUpModal.tsx

```diff
@@ -1,37 +1,121 @@
-import React, { FunctionComponent, useState } from 'react';
+import './Modal.scss';
+import { OverlayTrigger, Popover } from 'react-bootstrap';
 import CommonModal from '../components/organisms/Modal';
+import { createNewAccount } from '../services/auth';
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import {useState} from 'react';
 
-const SignUpModal: FunctionComponent = () => {
+function SignUpModal() {
```

Comment:
btw i think we should keep `const SignUpModal: FunctionComponent = () => {`, since this seems to be the standard if you look at the other components :)